internal/templates/source/callbacks/src/index.js

```diff
@@ -25,7 +25,9 @@ setLogLevel(argv["loglevel"] || "info");
   );
 
   ctx.register(ClassicLoader);
-  ctx.register(Classic());
+  ctx.register(Classic({
+    disableBatchAutoend: false
+  }));
```

Comment:
I don't think we need include the default here. We could include all the defaults, but as we add more options, this list will always be incomplete. I think we should instead have decent documentation about this.